### PR TITLE
CHORE: merge MyGet release into CI build

### PR DESCRIPTION
**Pull request by stijnmoreels**
Merge MyGet release into CI build.

Relates to #71

### CODE REVIEW

**Comment by stijnmoreels** [comment]
Integration tests fails with service bus exception?
https://dev.azure.com/codit/Arcus/_build/results?buildId=38508&view=results

File: build/ci-build.yml [hunk 1]
```diff
@@ -31,6 +27,30 @@ stages:
         pool:
           vmImage: 'ubuntu-16.04'
         steps:
+          - powershell: |
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Can we use this task? https://github.com/arcus-azure/azure-devops-templates/blob/master/nuget/determine-pr-version.yml

File: build/ci-build.yml [hunk 2]
```diff
@@ -1,18 +1,13 @@
 name: $(date:yyyyMMdd)$(rev:.r)
 
-trigger:
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Let's re-add this as we need them

**Comment by tomkerkhove** [comment]
That is correct, I think they've rolled out a change that broke this.

You have to change https://github.com/arcus-azure/arcus.eventgrid/blob/master/src/Arcus.EventGrid.Testing/Infrastructure/Hosts/ServiceBus/ServiceBusEventConsumerHost.cs#L144 to use `Accept-All` instead.

Thought I've sent a PR for this since I've fixed it recently but guess not 😅

**Comment by tomkerkhove** [comment]
@stijnmoreels Can you look at the failing test please?

**Comment by tomkerkhove** [comment]
@stijnmoreels Re-triggered it, let's see if it's due to flakyness or not.

File: build/ci-build.yml [hunk 3]
```diff
@@ -31,6 +39,9 @@ stages:
         pool:
           vmImage: 'ubuntu-16.04'
         steps:
+          - template: 'nuget/determine-pr-version.yml@templates'
+            parameters:
+              manualTriggerVersion: $(Package.Version.ManualTrigger)
           - task: DotNetCoreInstaller@0
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Let's use the `	build-solution.yml` here

File: build/ci-build.yml [hunk 4]
```diff
@@ -3,16 +3,23 @@ name: $(date:yyyyMMdd)$(rev:.r)
 trigger:
   branches:
     include:
-    - master
+      - master
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Is this intentionally?

File: build/ci-build.yml [hunk 5]
```diff
@@ -3,16 +3,23 @@ name: $(date:yyyyMMdd)$(rev:.r)
...
     include:
-    - src/*
+      - src/*
 
 pr:
   paths:
     include:
-    - src/*
-    - build/ci-build.yml
+      - src/*
```
Comment (tomkerkhove) [inline build/ci-build.yml]: Same here, intentionally?

File: build/ci-build.yml (hunk 4 above)
Comment (stijnmoreels) [reply build/ci-build.yml]: Hmm, format of vscode. I'll revert.

**Pull request merged**